hw/drivers/pwm/pwm_nrf52/src/pwm_nrf52.c

```diff
@@ -43,7 +43,7 @@ struct nrf53_pwm_dev_global {
     bool playing;
     nrf_drv_pwm_t drv_instance;
     nrf_drv_pwm_config_t config;
-    nrf_pwm_values_individual_t *duty_cycles;
+    nrf_pwm_values_individual_t duty_cycles[4];
```

Comment:
Should this be NRF_PWM_CHANNEL_COUNT, instead of 4?

Reply:
Fixed, thanks.